frigate/http.py

```diff
@@ -651,8 +652,15 @@ def latest_frame(camera_name):
         frame = current_app.detected_frames_processor.get_current_frame(
             camera_name, draw_options
         )
-        if frame is None:
-            frame = np.zeros((720, 1280, 3), np.uint8)
+
+        if frame is None or datetime.now().timestamp() > (
+            current_app.detected_frames_processor.get_current_frame_time(camera_name)
+            + 10
+        ):
+            error_image = glob.glob("/opt/frigate/frigate/images/camera-error.jpg")
+
+            if len(error_image) > 0:
+                frame = cv2.imread(error_image[0], cv2.IMREAD_UNCHANGED)
```

Comment:
Given the frequency that the latest.jpg endpoint is called in the debug view. It's probably better to read this once from the file system on startup rather than on each call.